tests/headers/complex.h

```diff
@@ -1,13 +1,15 @@
 
-#define COMPLEX_TEST(ty_)     \
-  struct Test##ty_ {          \
-    ty_ _Complex mMember;     \
-                              \
-  };                          \
-  struct Test##ty_##Ptr {     \
-    ty_ _Complex* mMember;    \
+#define COMPLEX_TEST(ty_, name_)  \
+  struct Test##name_ {            \
+    ty_ _Complex mMember;         \
+                                  \
+  };                              \
+  struct Test##name_##Ptr {       \
+    ty_ _Complex* mMember;        \
   };
 
-COMPLEX_TEST(double)
-COMPLEX_TEST(float)
-COMPLEX_TEST(int)
+COMPLEX_TEST(double, Double)
+COMPLEX_TEST(float, Float)
+
+// Not supported:
```

Comment:
Can you mention here, instead of that, something like:

```
// FIXME: 128-byte-aligned in some machines, which we can't support right now in Rust.
```